imgui-SFML.cpp

```diff
@@ -141,8 +128,34 @@ struct StickInfo {
     float threshold;
 };
 
-StickInfo s_dPadInfo;
-StickInfo s_lStickInfo;
+struct WindowContext {
+    sf::Window* window = nullptr;
```

Comment:
Change "nullptr" to "NULL" everywhere. Still supporting C++03, even if I don't really care about it much